pyiron_base/project/generic.py

```diff
@@ -222,8 +222,9 @@ def conda_environment(self):
             raise ImportError(
                 "You need to have the conda python package installed to access conda environments."
             ) from None
-
-        return CondaEnvironment()
+        conda_dir = os.path.join(self.path, "conda")
+        os.makedirs(conda_dir, exist_ok=True)
+        return CondaEnvironment(env_path=conda_dir)
```

Comment:
Would it not be possible to make the conda path configurable here, that would side step @srmnitc concerns, which I think are valid.